src/server/game/Handlers/DuelHandler.cpp

```diff
@@ -35,9 +35,16 @@ void WorldSession::HandleDuelAcceptedOpcode(WorldPacket& recvPacket)
     sLog->outStaticDebug("Player 2 is: %u (%s)", plTarget->GetGUIDLow(), plTarget->GetName().c_str());
 #endif
 
+	if(player->duel->underDuel || plTarget->duel->underDuel)
+	{
+		return;
+	}
+	
```

Comment:
please fix indentation

Reply:
They are ok in my file, I use notepad++ xD and when you open the file "View file" in Github that's ok. I cannot solve that with my IDE can u do it? :D

Reply:
switch your notepad from tabs to 4 space )

Reply:
or give me acces to your repo i fix it for you

Reply:
Repo already removed =)))